proto/src/serializers/bytes.rs

```diff
@@ -68,14 +68,14 @@ pub mod base64string {
     }
 }
 
-/// Serialize into Vec<base64string>, deserialize from Vec<base64string>
+/// Serialize into `Vec<base64string>`, deserialize from `Vec<base64string>`
```

Comment:
To avoid pseudo type references and generalize over serde formats:

```suggestion
/// Serialize into and deserialize from a sequence of _base64string_.
```